web/packages/teleport/src/services/api/api.ts

```diff
@@ -18,18 +18,36 @@
 
 import 'whatwg-fetch';
 
-import auth, { MfaChallengeScope } from 'teleport/services/auth/auth';
 import websession from 'teleport/services/websession';
 
+import 'whatwg-fetch';
```

Comment:
There is already an import for `whatwg-fetch`

Reply:
Weird that this isn't automatically caught by the auto-imports. When I get import merge conflicts usually just accepting both and re-saving the file fixes it.